pkgs/by-name/bu/bugstalker/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, libunwind
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bugstalker";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "godzie44";
+    repo = "BugStalker";
+    rev = "v${version}";
+    hash = "sha256-16bmvz6/t8H8Sx/32l+fp3QqP5lwi0o1Q9KqDHqF22U=";
+  };
+
+  cargoHash = "sha256-kp0GZ0cM57BMpH/8lhxevnBuJhUSH0rtxP4B/9fXYiU=";
+
+  buildInputs = [ libunwind ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  doCheck = false;
```

Comment:
Please add a comment explaining why to disable tests.

If some tests can be enabled, use

```
  checkFlags = [
    # reason for disabling test
    "--skip=example::tests:example_test"
  ];
```

Reply:
Many tests fail in the sandbox, even though they pass without a problem in a normal development environment. I haven't managed to work out how to inspect what is going on in the sandbox interactively, so the iteration time of almost 3 minutes makes it prohibitively expensive to get much more insight into it.

The errors all amount to "file not found" without giving any hint as to what file was not found. I suspect they are source files in the `examples` directory. Without an interactive means of working in the sandbox, I haven't managed to figure out how this might be remedied.